manufacturing/1.4/modules/ROOT/pages/release-notes.adoc

```diff
@@ -0,0 +1,42 @@
+= Release Notes
+:mfg-version: 1.4
+
+== v1.4
+
+*October 25, 2023*
+
+The Run-rate Business Management use case is updated to remove the dependency on MDM and optimize it for direct integration between ERP and CRM systems, such as connecting SAP S/4HANA data to Salesforce Manufacturing Cloud. As part of this update, the use case name is changed to https://anypoint.mulesoft.com/exchange/org.mule.examples/mulesoft-accelerator-for-manufacturing/minor/{mfg-version}/pages/Use%20case%201%20-%20Account%20and%20product%20and%20order%20sync/[Account and product and order sync^].
```

Comment:
Does it need to be Run-Rate? I could be wrong about capitalization here. 

Also, I assume this is the new name of the use case:
Account and product and order sync

Does it need to be in quotes since it's a name?